deployment/custom-deployment/lib/cdk-textract-stack.ts

```diff
@@ -223,17 +223,17 @@ export class CdkTextractStack extends cdk.Stack {
 
     // SQS queues
     const syncJobsQueue = new sqs.Queue(this, this.resourceName("SyncJobs"), {
-      visibilityTimeout: cdk.Duration.seconds(30),
+      visibilityTimeout: cdk.Duration.seconds(900),
       retentionPeriod: cdk.Duration.seconds(1209600)
     });
 
     const asyncJobsQueue = new sqs.Queue(this, this.resourceName("AsyncJobs"), {
-      visibilityTimeout: cdk.Duration.seconds(30),
+      visibilityTimeout: cdk.Duration.seconds(120),
```

Comment:
2 minutes is still a bit tight maybe?

Reply:
Async job queue just invokes the async handler only sends the job into Textract Async API. The chunk of the work is done by "Job Results Handler" which has a limit for 15 mins.